.travis.yml

```diff
@@ -18,11 +18,11 @@ compiler:
 notifications:
     email:
         recipients:
-            - su2code-dev@lists.stanford.edu
+            - pcarruscag@gmail.com
   
 branches:
     only:
-        - develop
+        - merge_cvariable_output
```

Comment:
How do I make travis run on this branch?

Reply:
Set it to feature_input_output, I think that should. Otherwise we have to manually trigger travis for now.